src/MDMFadeTransition.m

```diff
@@ -16,7 +16,7 @@
 
 #import "MDMFadeTransition.h"
 
-@import MotionAnimator;
+#import "MotionAnimator.h"
```

Comment:
Two questions:
* Does `#import` of a plain header have the same benefits of `@import`ing the module?
* If `MotionAnimator.h` is an umbrella header for a different library, should we instead use the form:
```objectivec
#import <MotionAnimator/MotionAnimator.h> 
```

Reply:
The primary motivation for moving away from @import statements is to support bazel builds, which have a different opinion about how `@import` modules should be named.

Re: `<Umbrella/Umbrella.h>` or `"Umbrella.h"` form, either is ok though we've been using `"Umbrella.h"` in MDC-iOS pretty consistently. I've run into build errors attempting to use the `"Umbrella.h"` variant in headers but not in source files in this project and I'm not entirely certain as to why. Will have to look into that in more detail.

Reply:
Ack.